app/admin/user.rb

```diff
@@ -335,4 +339,28 @@ def create
       redirect_to admin_user_path, notice: I18n.t('resend_user_registration_confirmed')
     end
   end
+
+  member_action :export_experience, method: :post do
+    user = User.find(params[:id])
+    doc = UserResumeDocService.new
+
+    doc.add_user_info(user)
+
+    skills = user.skills.pluck(:title)
+    doc.mount_skill_section(skills)
+
+    job_experiences = user.professional_experiences.order(end_date: :asc).decorate
+    doc.mount_professional_section(job_experiences)
+
+    education_experiences = user.education_experiences.decorate
+    doc.mount_education_section(education_experiences)
+
+    contribution_experiences = user.contributions.approved.order(created_at: :asc).decorate
+    doc.mount_contribution_section(contribution_experiences)
+
+    talk_experiences = user.talks.decorate
+    doc.mount_talk_section(talk_experiences)
```

Comment:
this could be in a external class

Reply:
should it also go in the `UserResumeDocService`?

Reply:
no, UserResumeDoc, it should not be a service, but ok.
User resume doc describes the word doc, and we should have a class that builds the doc

Reply:
done